### PR TITLE
Task #210374 [WEB-ADMIN] Improving the UI of Preark list (Dashboard) and Prerak profile view

### PR DESCRIPTION
**Pull request by chaitanyakole**
### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [x] Code is formatted as per format decided
* [x] Updated acceptance criteria in tracker
* [NA] Updated test cases in test-cases-tracker
https://tracker.tekdi.net/issues/210374

image

https://github.com/tekdi/eg-website/assets/124124878/b6f24842-0b77-4076-86b0-f697b9b4b94d

## Summary by CodeRabbit

- **New Features**
	- Added the `Select`, `CheckIcon`, and `Stack` components to enhance facilitator page layout.
	- Introduced the `status` field for improved facilitator management.
	- Implemented a new drawer feature to provide additional UI options.

- **Bug Fixes**
	- Refined logic for status field and filter state handling to ensure accurate display and functionality.

- **Refactor**
	- Removed the unused `CustomRadio` component for cleaner code.
	- Replaced ScrollView with VStack for a more efficient status filter display.
	- Streamlined rendering of status buttons with conditional logic for improved performance.

- **Style**
	- Updated typography styles for better visual hierarchy and emphasis.
	- Introduced `CardComponent` for a more organized presentation of profile details.

### CODE REVIEW

File: apps/front-end/src/pages/admin/facilitator/View.js [hunk 1]
```diff
@@ -584,288 +585,147 @@ export default function FacilitatorView({ footerLinks }) {
...
+                      data?.device_ownership.slice(1),
+                    device_type:
+                      data?.device_type.charAt(0).toUpperCase() +
+                      data?.device_type.slice(1),
+                  })}
+                  arr={["availability", "device_ownership", "device_type"]}
+                  // onEdit={(e) => navigate(`/beneficiary/edit/${id}/contact-info`)}
+                />
               </VStack>
             </HStack>
```
[the remark this answers is not in the document]
Comment (chaitanyakole) [reply apps/front-end/src/pages/admin/facilitator/View.js:730]: Not required

File: apps/front-end/src/pages/admin/facilitator/Table.js [hunk 2]
```diff
@@ -203,47 +203,26 @@ function Table({
...
+            filter?.status
+              ?.filter((item) => item)
+              .map((item) => t(item).toLowerCase())
+              .map((item) => item.charAt(0).toUpperCase() + item.slice(1))
+              .join(" , ")
+            //+
+            // `(${paginationTotalRows})` +
+            // " "
+          )}
+        </AdminTypo.H5>
```
[the remark this answers is not in the document]
Comment (chaitanyakole) [reply apps/front-end/src/pages/admin/facilitator/Table.js:224]: not required

File: apps/front-end/src/pages/admin/facilitator/List.js [hunk 3]
```diff
@@ -508,61 +517,117 @@ export default function List({ footerLinks, userTokenInfo }) {
...
-                borderColor="#eee"
-                flexWrap="wrap"
+            <Box
+              flex={[2, 2, 1]}
+              style={{
+                borderRightColor: "dividerColor",
+                borderRightWidth: "2px",
+              }}
+            >
+              <HStack ref={ref}></HStack>
```
Comment (swapnilphalke03) [inline apps/front-end/src/pages/admin/facilitator/List.js]: check this if required

**Review by swapnilphalke03 — CHANGES_REQUESTED** [review]

File: apps/front-end/src/pages/admin/facilitator/List.js [hunk 4]
```diff
@@ -508,61 +508,117 @@ export default function List({ footerLinks, userTokenInfo }) {
...
+              filter?.block ||
+              filter?.qualificationIds ||
+              filter?.status
+                ? "white"
+                : "black"
+            }
+            _icon={{ size: "30px" }}
+          />
+        </VStack>
+
```
[the remark this answers is not in the document]
Comment (chaitanyakole) [reply apps/front-end/src/pages/admin/facilitator/List.js]: Not required

**Review by swapnilphalke03 — APPROVED** [review]

**Pull request closed without merging**